payments/database.py

```diff
@@ -6,7 +6,7 @@ def create_database(name="database.db"):
     with sqlite3.connect(name) as conn:
         logging.info("Creating new {}...".format(name))
         conn.execute(
-            "CREATE TABLE payments (uuid TEXT, fiat_value DECIMAL, btc_value DECIMAL, method TEXT, address TEXT, time DECIMAL, webhook TEXT, rhash TEXT)"
+            "CREATE TABLE payments (uuid TEXT, base_value DECIMAL, btc_value DECIMAL, method TEXT, address TEXT, time DECIMAL, webhook TEXT, rhash TEXT)"
```

Comment:
This will not work when upgrading, should instead rename `fiat_value` to `base_value` below.
```suggestion
            "CREATE TABLE payments (uuid TEXT, fiat_value DECIMAL, btc_value DECIMAL, method TEXT, address TEXT, time DECIMAL, webhook TEXT, rhash TEXT)"
```

Reply:
fixed, ready to merge i think